exporters/trace/src/main/java/com/google/cloud/opentelemetry/trace/InternalTraceExporter.java

```diff
@@ -53,7 +56,10 @@ class InternalTraceExporter implements SpanExporter {
   private final TraceTranslator translator;
 
   private static final Map<String, String> HEADERS =
-      Map.of("User-Agent", "opentelemetry-operations-java/" + TraceVersions.EXPORTER_VERSION);
+      Stream.of(
```

Comment:
Could use `singletonMap()` here? https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#singletonMap-K-V-

Reply:
Good catch, yes - I could replace it.